spec/controllers/api/v1/mappings_controller_spec.rb

```diff
@@ -8,7 +8,7 @@
   context 'on GET to index' do
     context 'requesting json representation', api_specification: true do
       let(:mappings_schema) do
-        schema_for('ontology/commands/mappings')
+        schema_for_command('ontology/mappings', :get, 200)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.